src/models/data.py

```diff
@@ -298,6 +301,13 @@ def _calculate_historical(self, x: xr.Dataset,
                 list(x.data_vars))
             x_np = (x_np - self.normalizing_array['mean']) / (self.normalizing_array['std'])
 
+        if self.months_from_pred:
```

Comment:
so this is the extra column which tells the model `which timestep it is looking at `?

what do you mean by timestep? An integer from 0-N for each month from 1981-2019?

That would account for a 'global warming' kinda signal to be fair